corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -236,6 +237,51 @@ public int dropTable(String namespace, String tablename) {
         return tableSize;
     }
 
+    /**
+     * Delete a specific record in a table using the JSON output from printTable
+     * @param namespace - the namespace where the table belongs
+     * @param tablename - table name without the namespace
+     * @param deleteRecordKey - the JSON key of the record to be deleted
+     * @return - number of entries in the table after deleting the record
+     */
+    public int deleteRecord(String namespace, String tablename, String deleteRecordKey) {
+        verifyNamespaceAndTablename(namespace, tablename);
+        log.info("\n======================\n");
+        String fullName = TableRegistry.getFullyQualifiedTableName(namespace, tablename);
+        UUID streamUUID = UUID.nameUUIDFromBytes(fullName.getBytes());
+        CorfuTable<CorfuDynamicKey, CorfuDynamicRecord> table = getTable(namespace, tablename);
+
+        TableName tableNameProto = TableName.newBuilder().setNamespace(namespace).setTableName(tablename).build();
+        final CorfuStoreMetadata.TableDescriptors tableDescriptor = runtime.getTableRegistry().getTableDescriptor(tableNameProto);
+        if (tableDescriptor == null) {
+            return 0;
+        }
+        final DynamicMessage.Builder builder = DynamicMessage.newBuilder(tableDescriptor);
+        try {
+            JsonFormat.parser().merge(deleteRecordKey, builder);
```

Comment:
I am surprised how this will work... the schema of 'deleteRecordKey' and 'builder' is different